architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,81 +350,146 @@ metadata:
-retrieve the images tagged in a project they have access to using this resource.
+To refer to the image in the
+xref:image-stream-object-definition[image stream object example above], the image stream image looks like:
+
+----
+origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+----
 
 [[image-stream-tag]]
-=== Image Stream Tag
+=== Image Stream Tags
 
 An _image stream tag_ is a named pointer to an image in an _image stream_. It
-is often abbreviated as _istag_. It can reference any local or externally
+is often abbreviated as _istag_. An image stream tag is used to reference or retrieve
+an image for a given image stream and tag.
+
+Image stream tags can reference any local or externally
 managed image. It contains a history of images represented as a stack of all
 images the tag ever pointed to. Whenever a new or existing image is tagged
-under particular _istag_, it is just placed at the first position in the
-history stack. Image previously occupying the top position will be available at
-the second position etc. This allows for easy rollbacks to make tags point to
+under particular image stream tag, it is placed at the first position in the
+history stack. The image previously occupying the top position will be available at
+the second position, and so forth. This allows for easy rollbacks to make tags point to
 historical images again.
 
-The _istag_ is composed of two parts separated by a colon: `<image stream
-name>:<tag>`. _Istag_ referring to the image
-`sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d` in
-the xref:image-stream-object-definition[example above] would be
-`origin-ruby-sample:latest`.
+Image stream tags can be _Permanent_ tags or _tracking_ tags.
+
+* _Permanent tags_ are version-specific tags that point to a particular version of an image, such as Python 3.5.
+
+* _Tracking tags_ are reference tags that follow another image stream tag and could be updated in the future to change which image they follow, much like a symlink. Note that these new levels are not guaranteed to be backwards-compatible.
++
+For example, the `latest` image stream tags that ship with {product-title} are tracking tags. This means consumers of the `latest` image stream tag will be updated to the newest level of the framework provided by the image when a new level becomes available. A `latest` image stream tag to v3.6 could be changed to v3.7 at any time. It is important to be aware that the `latest` image stream tag behaves differently than the Docker `latest` tag. The `latest` image stream tag does not point to the latest image in the Docker repository. It points to another image stream tag that might not be the latest version of an image. For example, if the `latest` image stream tag points to v3.2 of an image, when the 3.3 version is released, the `latest` tag is not automatically updated to v3.3, and remains at v3.2 until it is manually updated.  
```

Comment:
> The `latest` image stream tag does not point to the latest image in the Docker repository.

The `latest` image stream tag does not, *in this case,* point to the latest image in the Docker repository.

Reply:
> It is important to be aware that the `latest` image stream tag behaves differently than the Docker `latest` tag.

It is important to be aware that *these* `latest` image stream *tags* behaves differently than the Docker `latest` tag.

Reply:
> For example, if the `latest` image stream tag points to v3.2 of an image, when the 3.3 version is released, the `latest` tag is not automatically updated to v3.3, and remains at v3.2 until it is manually updated.  

For example, if the `latest` image stream tag points to an image stream tag for v3.2 of an image, when the 3.3 version is released, the `latest` tag is not automatically updated to v3.3, and remains at v3.2 until it is manually updated to point to a v3.3 image stream tag.